.github/workflows/link-check-deployment-status.yml

```diff
@@ -2,11 +2,11 @@ name: Check new links against deployment
 on:
   workflow_call:
     inputs:
-      main:
+      branch:
         type: "string"
         required: false
-        description: "The main branch of this repository"
-        default: "main"
+        description: "The branch to compare against when finding new links to check"
+        default: "${{ github.event.repository.default_branch }}"
```

Comment:
according to https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability this should actually work fine :)

Reply:
I was skeptical at first as well and was ready to just axe the option, but I made sure to test this out first and it surprisingly went just fine.